server/tests/metadata.rs

```diff
@@ -24,6 +24,53 @@ mod entities_metadata_query {
                     "LearningResource",
                     "Article"
                   ],
+                  "creator": [
```

Comment:
To fix the tests: Make the order deterministic (e.g. Order by username alphabetically or order by user.id)